### DISCUSSION TITLE
欢迎来到 讨论区！

### DISCUSSION THREAD

**Discussion by PlexPt** [PlexPt/awesome-chatgpt-prompts-zh#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bqbncjx** [comment]
英语自我介绍

**Comment by bqbncjx** [comment]
write introduce myself

**Comment by vinson-lee01** [comment]
hello

**Comment by kjmn3** [comment]
what is it？
